glib/src/gobject/flags.rs

```diff
@@ -3,13 +3,20 @@
 use crate::translate::*;
 
 bitflags::bitflags! {
+    #[doc(alias = "GParamFlags")]
     pub struct ParamFlags: u32 {
-        const READABLE = 1;
-        const WRITABLE = 2;
-        const READWRITE = 3;
-        const CONSTRUCT = 4;
-        const CONSTRUCT_ONLY = 8;
-        const LAX_VALIDATION = 16;
+        #[doc(alias = "G_PARAM_READABLE")]
+        const READABLE = gobject_ffi::G_PARAM_READABLE as u32;
+        #[doc(alias = "G_PARAM_WRITABLE")]
+        const WRITABLE = gobject_ffi::G_PARAM_WRITABLE as u32;
+        #[doc(alias = "G_PARAM_READWRITE")]
+        const READWRITE = gobject_ffi::G_PARAM_READWRITE as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT")]
+        const CONSTRUCT = gobject_ffi::G_PARAM_CONSTRUCT as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT_ONLY")]
+        const CONSTRUCT_ONLY = gobject_ffi::G_PARAM_CONSTRUCT_ONLY as u32;
+        #[doc(alias = "G_PARAM_LAX_VALIDATION")]
+        const LAX_VALIDATION = gobject_ffi::G_PARAM_LAX_VALIDATION as u32;
         const USER_0 = 128;
```

Comment:
Actually `USER_0` makes no sense. It starts at `1<<8`, which is `USER_1 = 256`

Reply:
Removed.